include/aos/common/config.hpp

```diff
@@ -519,6 +519,13 @@
 #define AOS_CONFIG_TYPES_FUNC_SERVICE_MAX_COUNT 16
 #endif
 
+/**
+ * Max number of outdated items.
+ */
+#ifndef AOS_CONFIG_TYPES_MAX_OUTDATED_ITEMS
```

Comment:
Rename to AOS_CONFIG_SPACEALLOCATOR_MAX_OUTDATED_ITEMS

Reply:
Done